astroquery/__init__.py

```diff
@@ -27,3 +27,26 @@ def _get_bibtex():
 
 
 __citation__ = __bibtex__ = _get_bibtex()
+
+# backward compatibility: retain the flat namespace as of May 2019 at least
+# until we deprecate it
+
+# categories of data type
+from atomic_molecular_databases import atomic, hitran, jplspec, nist, splatalogue, vamdc
+from simulations import besancon, cosmosim
+from observatory_archives import alma, fermi, gaia, lcogt, nrao
+from image_cutout_services import skyview, magpis
+
+# institutes with multiple submodules
+from wfau import vsa, ukidss
+from cds import simbad, vizier, xmatch
+from ipac import ibe, irsa, irsa_dust, sha, ned
+from esa import esasky, gaia, hubble
```

Comment:
`hubble` should not be imported but only available as `esa.hubble`, the whole restructuring needs to be done in 0.3.10 to avoid having two hubble modules in the main namespace.